From 3769f6c6c393a62a33a3b7c680943d3ddc70eaeb Mon Sep 17 00:00:00 2001
From: James Slagle <jslagle@redhat.com>
Date: Sun, 23 Mar 2014 16:53:01 -0400
Subject: [PATCH] Add missing +x

Adding another missed +x on a hook script. Luckily diblint is on the
way.

Change-Id: Ic6ef80137821ee14cdf57e87c17a8a2f3950c447
---
 .../install.d/os-collect-config-package-install/10-os-collect-config      | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 elements/os-collect-config/install.d/os-collect-config-package-install/10-os-collect-config

diff --git a/elements/os-collect-config/install.d/os-collect-config-package-install/10-os-collect-config b/elements/os-collect-config/install.d/os-collect-config-package-install/10-os-collect-config
old mode 100644
new mode 100755
-- 
1.8.5.3

